content/blog/2025/01/08/2025-01-08-jenkins-2024-recap.adoc

```diff
@@ -0,0 +1,176 @@
+He shared details about how incorporating Docker Compose makes the tutorial experience more simplified and beneficial to the user overall, and how it is safer than using the previous Docker-in-Docker method.
+Users benefit from both the comfort of knowing their build is secure and a more straightforward example, allowing them to get started faster.
+
+The last event of the year that involved Jenkins was an link:https://techstrong.tv/videos/cd-pipeline/behind-the-code-jenkins-upgrades-challenges-and-evolution-the-cd-pipeline-ep15[episode of Techstrong TV] that featured author:markewaite[Mark Waite], author:basil[Basil Crow], author:dduportal[Damien Duportal], and author:krisstern[Kris Stern], where they discussed the challenges and successes of the ever-evolving Jenkins project.
+They discussed how upgrades and the constantly changing landscape of development influence the Jenkins project's direction and what it takes to manage a large project like Jenkins.
+Thanks to Techstrong TV for having Jenkins on and to the Jenkins representatives for sharing their insights and anecdotes about the project.
+
+[[modern-ui]]
+image:/images/post-images/2025/01/08/ui-ux.png[User Experience Update]
+
+There were also presentations from link:https://www.jenkins.io/blog/2024/08/29/jenkins-design/[Jenkins contributors at the virtual DevOps World event], showcasing some of the UI/UX updates and enhancements that have been implemented in the last year.
+This includes everything from buttons and menu items being refreshed, to more modern page design, and enhancements intended to make the Jenkins user experience more smooth and inviting overall.
+There have been additional UI updates since the talk, such as providing the help text within Jenkins directly, instead of hiding it in an extra button.
+This allowed us to showcase the work and effort that goes into refining the Jenkins experience even further.
+Thanks to author:janfaracik[Jan Faracik] and author:timja[Tim Jacomb] for delivering this talk and providing so many great updates to the Jenkins UI/UX.
+
+Additional thanks to all those who help enhance the Jenkins UX/UI through their efforts to provide the best experience possible.
+Constantly working to enhance the Jenkins user experience, the folks working on UI/UX have a passion for making Jenkins more sleek and modern.
+This passion translates to providing Jenkins users with a more friendly and enjoyable experience when using Jenkins, whether they're long-time users or just getting started.
+They have refreshed Jenkins menus and pages, link:https://www.jenkins.io/blog/2024/12/02/pr-titles/[incorporated user generated requests], and implemented several changes to refine the UI such as overhauling the search function.
```

Comment:
YUI is also gone which was a major effort

Reply:
Maybe a short mention of the [redesigned builds widget](https://github.com/jenkinsci/jenkins/pull/9148) and revamped [v3 of the Design Library](https://github.com/jenkinsci/design-library-plugin/pull/359).

Reply:
Both sound like good ideas, I'll add them to the UI/UX section along with the links provided here. Thank very much @timja and @janfaracik